x-pack/solutions/search/plugins/enterprise_search/public/applications/enterprise_search_content/components/connector_detail/connector_stats.tsx

```diff
@@ -315,6 +390,114 @@ export const ConnectorStats: React.FC<ConnectorStatsProps> = ({ connector, index
+                <EuiFlexItem grow={false}>
+                  <EuiButtonEmpty
+                    isDisabled={!connector.service_type}
+                    iconType={EuiIconPlugs}
+                    color="text"
+                    href={http.basePath.prepend(
+                      generateEncodedPath(CONNECTOR_INTEGRATION_DETAIL_PATH, {
+                        serviceType: connector.service_type || '',
+                      })
+                    )}
+                  >
+                    Elastic Connectors
+                  </EuiButtonEmpty>
+                </EuiFlexItem>
+                <EuiFlexItem grow={false}>
+                  {agentlessAgentExists ? (
+                    <AgentlessConnectorStatusBadge status={agentlessOverview?.agent.status} />
+                  ) : (
+                    <EuiBadge color="default">{noAgentLabel}</EuiBadge>
+                  )}
+                </EuiFlexItem>
+              </EuiFlexGroup>
+            }
+            footer={
+              <EuiFlexGroup justifyContent="spaceBetween" alignItems="center">
+                <EuiFlexItem grow={false}>
+                  <EuiButton
+                    data-test-subj="connectorStatsViewLogsButton"
+                    aria-label={i18n.translate(
+                      'xpack.enterpriseSearch.connectors.connectorStats.viewLogsButtonLabel',
+                      { defaultMessage: 'View logs' }
+                    )}
+                    disabled={!agentlessAgentExists}
+                    iconType="discoverApp"
+                    onClick={() => {
+                      discover.locator?.navigate(navigateToDiscoverPayload);
+                    }}
+                  >
+                    {i18n.translate(
+                      'xpack.enterpriseSearch.connectors.connectorStats.viewLogsButtonLabel',
+                      { defaultMessage: 'View logs' }
+                    )}
+                  </EuiButton>
+                </EuiFlexItem>
+                <EuiFlexItem grow={false}>
+                  {agentlessAgentExists ? (
+                    <EuiButtonEmpty
+                      isDisabled={!agentlessOverview || !agentlessOverview.agent.id}
+                      size="s"
+                      href={http.basePath.prepend(
+                        generateEncodedPath(FLEET_AGENT_DETAIL_PATH, {
+                          agentId: agentlessOverview.agent.id,
+                        })
+                      )}
+                    >
+                      {i18n.translate(
+                        'xpack.enterpriseSearch.connectors.connectorStats.hostOverview',
+                        {
+                          defaultMessage: 'Host overview',
+                        }
+                      )}
+                    </EuiButtonEmpty>
+                  ) : (
+                    <EuiText size="s" color="warning">
+                      {noAgentLabel}
+                    </EuiText>
```

Comment:
I think is redundant to display twice _No agent found_ I'd leave only the on in the badge. The footer should only contain actionable links
```suggestion
                    null
```

![CleanShot 2025-02-11 at 09 59 09@2x](https://github.com/user-attachments/assets/6029ce60-445e-4bc7-a7e5-e542e24ad7bd)